docs/subnets/how-to-import-public.md

```diff
@@ -0,0 +1,208 @@
+# How to Import a Subnet from a Public Network to `Avalanche-CLI`
+
+## What Is the Context of This How-To?
+
+The most probable reason why someone would want to do this, 
+is if they already deployed a Subnet with `subnet-cli` either to `Fuji` or `Mainnet`, 
+and now would like to use `Avalanche-CLI` to manage the Subnet.  
```

Comment:
We should add a lint about first installing avalanche-cli https://docs.avax.network/subnets/install-avalanche-cli

Reply:
Added